### ISSUE TITLE
Error inconsistent result after apply with vercel_project

### ISSUE THREAD

**Issue by mdenkinger** [vercel/terraform-provider-vercel#187]
I am having problems with the vercel_project. Here is the full error message:
```
  # vercel_project.default-project_779E12D6 (default-project/project) will be updated in-place
  ~ resource "vercel_project" "default-project_779E12D6" {
        id                                                = "prj_1y64fxxxx"
        name                                              = "xxxx"
      + password_protection                               = {
          + deployment_type = "all_deployments"
          + password        = (sensitive value)
        }
      + protection_bypass_for_automation_secret           = (known after apply)
      + trusted_ips                                       = {
          + addresses       = (sensitive value)
          + deployment_type = "all_deployments"
          + protection_mode = "trusted_ip_required"
        }
        # (14 unchanged attributes hidden)
    }
Plan: 0 to add, 1 to change, 0 to destroy.
vercel_project.default-project_779E12D6: Modifying... [id=prj_1y64fxxx]
╷
│ Error: Provider produced inconsistent result after apply
│ 
│ When applying changes to vercel_project.default-project_779E12D6 (default-project/project), provider
│ "provider[\"registry.terraform.io/vercel/vercel\"]" produced an unexpected
│ new value: .trusted_ips.addresses: planned set element
│ cty.ObjectVal(map[string]cty.Value{"note":cty.NullVal(cty.String),
│ "value":cty.StringVal("xx.xxx.xxx.xxx")}) does not correlate with any element
│ in actual.
│ 
│ This is a bug in the provider, which should be reported in the provider's own
│ issue tracker.
```

I've faced the issue currently with `trusted_ips`, where it persists.   I have seen it also for `environment` with the same provider version.

I am using this terraform configuration for this:
```
 "terraformProviders": [
    "vercel/vercel@~> 1.11"
  ]
```

**Comment by mdenkinger** [comment]
Adding some `note` to the trusted ip addresses fixes the issue as a workaround.  And sure, adding notes is a best practices :) 
```
export interface ProjectTrustedIpsAddresses {
  /**
  * A description for the value
  *
  * Docs at Terraform Registry: {@link https://registry.terraform.io/providers/vercel/vercel/1.11.0/docs/resources/project#note Project#note}
  */
  readonly note?: string;
  /**
  * The address or CIDR range that can access deployments.
  *
  * Docs at Terraform Registry: {@link https://registry.terraform.io/providers/vercel/vercel/1.11.0/docs/resources/project#value Project#value}
  */
  readonly value: string;
}
```

**Comment by dglsparsons** [comment]
Hi, sorry you're experiencing this. It's a provider bug - we will look into getting it sorted asap.

**Comment by kitfoster** [comment]
Thanks for raising @mdenkinger 🙏 this is fixed in [v1.11.1](https://github.com/vercel/terraform-provider-vercel/releases/tag/v1.11.1)
